ud/ReadMe

```diff
@@ -0,0 +1,9 @@
+This ud directory is for personal use, as user directory
```

Comment:
What do you mean personal use?

I see a bunch of namelist files here. What is this for?

Reply:
I used ud/ to store my tests for those scripts and nml George provided. I don't need to merge that to develop after regress test for wam is there.

Reply:
Let me do whatever I can and we can have  iteratively progress. I will commit and push, then please check then, many thanks.